website/docs/reference/supported_commands.md

```diff
@@ -11,3 +11,51 @@ sidebar_position: 1
 |                   | `nameOnly`              | ❌     | [Issue](https://github.com/FerretDB/FerretDB/issues/301)  |
 |                   | `comment`               | ⚠️     | Ignored                                                   |
 |                   | `authorizedCollections` | ⚠️     | Ignored                                                   |
+
+## Aggregation pipelines
+
+The epic - [Issue](https://github.com/FerretDB/FerretDB/issues/9).
+
+| Command     | Argument             | Status | Comments                                                  |
+|-------------|----------------------|--------|-----------------------------------------------------------|
+| `aggregate` |                      | ❌      | [Issue](https://github.com/FerretDB/FerretDB/issues/1410) |
```

Comment:
Won't it be better to have this placed together with the other commands above?

Reply:
Good idea too. @AlekSi can take a look?

Reply:
We could do it later if needed